unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/Variables/BaseTriggerVariable.cs

```diff
@@ -0,0 +1,23 @@
+﻿/*
+ * BaseTriggerVariable raise OnChange event everytime it value is set
+ * regardless if the value is the same as it previous value.
+ */
+
+public class BaseTriggerVariable<T> : IBaseVariable<T>
```

Comment:
Not sure about this class name (`BaseTriggerVariable`)
I accept suggestion to change it :P

Reply:
BaseEventVariable?  A tricky one to name